reinforcement_learning/common/sagemaker_rl/ray_launcher.py

```diff
@@ -123,7 +123,7 @@ def get_all_host_names(self):
 
     def ray_init_config(self):
         num_workers = max(self.num_cpus, 3)
-        config = {"num_cpus": num_workers, "num_gpus": self.num_gpus, "webui_host": '127.0.0.1'}
```

Comment:
Any specific reason to remove `webui_host`? iirc this will suppress the misleading ray dashboard error.

Reply:
This is to make sure that the examples run using Ray 0.6.5, which is the one supported by SDK v1.72. `webui_host` seems to have been introduced to the init function in later versions of Ray.